packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function removeAddEventListener(file, api) {
```

Comment:
From snooping at esified code, removing `return` from nested statements would be useful.  But it's probably best to do it in a separate transform.